src/lib/addons/addon.ts

```diff
@@ -75,15 +71,13 @@ export default abstract class Addon {
             this.logger.warn(
                 `Error querying ${url} with method ${
                     method || 'GET'
-                } status code ${e.code}`,
+                } status code ${(e as any).code}`,
                 e,
             );
-            res = { status: e.code, ok: false };
+            return { status: e.code, ok: false } as Response;
```

Comment:
Okay this cast is janky. But I see why we need it and it's not making things worse, we already had an implicit any here